lib/http-outgoing.js

```diff
@@ -146,6 +150,24 @@ export default class PodletClientHttpOutgoing extends PassThrough {
         this.#redirectable = redirectable;
     }
 
+    get js() {
+        // return the internal js value or, fallback to the manifest for backwards compatibility
+        return this.#js || this.#manifest.js;
```

Comment:
Should the manifest property be prefixed with underscore? I see it is in the fallback.

```suggestion
        return this.#js || this.#manifest._js;
```

Reply:
No, what's happening here is that our new feature stores js on this.#js (both on content or fallback) and if present (ie, the podlet sent early hints), we return it. If it's not, we fallback to the old way of doing things which was grabbing the js value from the manifest file.